jax/random.py

```diff
@@ -199,12 +199,16 @@
     "PRNGKeyArray": (
         "jax.random.PRNGKeyArray is deprecated. Use jax.Array for annotations, and "
```

Comment:
Maybe here you can write: `jax.random.PRNGKeyArray is deprecated. Please use the new typed key arrays. ...`

Reply:
If people are referencing `PRNGKeyArray`, presumably they already know about typed PRNG keys, because this is the runtime type of typed PRNG keys.

Reply:
The only reason you'd use `jax.random.PRNGKeyArray` is in a type annotation or in an instance check, so I suggest replacements for those two use-cases.

Reply:
Well, looking at the chat thread, that wasn't true.

Reply:
That was using `KeyArray`, not `PRNGKeyArray`

Reply:
I suppose under `KeyArray` I could describe what typed PRNG keys are, but I think the link is sufficient for that – WDYT?

Reply:
Sounds good.

Reply:
i.e. the link is good enough since that explains everything.